src/schedule.rs

```diff
@@ -16,19 +16,42 @@ pub struct Checkpoint {
 pub struct Schedule(pub Vec<Checkpoint>);
 
 impl Schedule {
-    pub fn new_unlocked(total_balance: Balance) -> Self {
+    pub fn new_zero_balance_from_to(
+        start_timestamp: TimestampSec,
+        finish_timestamp: TimestampSec,
+    ) -> Self {
+        assert!(finish_timestamp > start_timestamp, "Invariant");
+
         Self(vec![
             Checkpoint {
-                timestamp: 0,
+                timestamp: start_timestamp,
                 balance: 0,
             },
             Checkpoint {
-                timestamp: 1,
+                timestamp: finish_timestamp,
+                balance: 0,
+            },
+        ])
+    }
+
+    pub fn new_unlocked_since(total_balance: Balance, timestamp: TimestampSec) -> Self {
+        assert!(timestamp > 0, "Invariant");
+        Self(vec![
+            Checkpoint {
+                timestamp: timestamp - 1,
+                balance: 0,
+            },
+            Checkpoint {
+                timestamp: timestamp,
                 balance: total_balance,
             },
         ])
     }
 
+    pub fn new_unlocked(total_balance: Balance) -> Self {
+        Self::new_unlocked_since(total_balance, 1)
+    }
```

Comment:
nit: Do we still need this method? I mean, without it, we can clearly say that we don't create timestamp = 1 by default anymore.

Reply:
Added conditional compilation of the method only for non-wasm builds in the line above.